ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_losses.py

```diff
@@ -76,3 +76,91 @@ def test_binary_cross_entropy_with_logits(
         pos_weight=pos_weight[0],
         reduction=reduction,
     )
+
+#mse
+@handle_test(
+    fn_tree="functional.ivy.experimental.mse_loss",
+    dtype_and_true=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+        max_num_dims=3,
+    ),
+    dtype_and_pred=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+        max_num_dims=3,
+    ),
+    reduction=st.sampled_from(["none", "sum", "mean"]),
+)
+def test_mse_loss(
+    dtype_and_true,
+    dtype_and_pred,
+    reduction,
+    test_flags,
+    backend_fw,
+    fn_name,
+    on_device,
+    ground_truth_backend,
+):
+    pred_dtype, pred = dtype_and_pred
+    true_dtype, true = dtype_and_true
+    # Add dimension handling
+    if len(pred.shape) > len(true.shape):
```

Comment:
As generated examples are lists you will need to unpack them

Reply:
@hirwa-nshuti - Thank you very much for your instant review :smile:  I am so sorry that I didn't run the pytest before committing. I shall do this and recommit with updated changes.

Request you to wait for some time :blush:  as I am a bit involved in some urgent tasks!

Thanks for your understanding.